python/ray/dashboard/modules/event/event_agent.py

```diff
@@ -90,7 +90,7 @@ async def report_events(self):
                 self.total_request_sent += 1
                 break
             except Exception as e:
-                logger.warning(f"Report event failed, retrying... {e}")
+                logger.warning("Report event failed, retrying...", exc_info=True)
```

Comment:
mm.. why not just use `logger.exception`?

Reply:
then it will be ERROR log, we want it to be WARNING since we retry it and if all retries fail, we have an error log later.